winml/dll/module.cpp

```diff
@@ -7,6 +7,10 @@
 
 #include "LearningModelDevice.h"
 #include "OnnxruntimeProvider.h"
+#include "Dummy.h"
+
+#define STRINGIFY(x) #x
+#define XSTRINGIFY(x) STRINGIFY(x)
```

Comment:
What's the purpose of defining this twice?

Reply:
C MACROs params are expanded in a particular order. to convert to string you need indirection of another macro.